<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplashScreen</class>
 <widget class="QWidget" name="SplashScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#SplashScreen{
	background-color: rgba(52,52,52);
	border: 1px solid black;
	border-radius: 60px;
}
QLabel#lb_title{
	color: qlineargradient(spread:reflect, x1:0, y1:0.506, x2:0, y2:1, stop:0 rgba(255, 233, 60, 255), stop:1 rgba(255, 249, 177, 255));;
}
QProgressBar{
	background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:0, y2:0.477, stop:0.114833 rgba(169, 169, 169, 255), stop:0.794258 rgba(220,220,220, 255));
	border: 1px solid white;
	border-radius: 8px;
	color: black;
}
QProgressBar:chunk{
	border: 0px solid white;
	border-radius: 6px;
} 
QProgressBar#pb_loadActors:chunk{
	background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:0.5, stop:0.114833 rgba(229, 122, 205, 255), stop:0.794258 rgba(199, 27, 166, 255));
}
QProgressBar#pb_loadScenes:chunk{
	background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:0.5, stop:0.114833 rgba(106, 180, 229, 255), stop:0.794258 rgba(25, 141, 199, 255));
}
QProgressBar#pb_buildScenes:chunk{
	background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:0.5, stop:0.119617 rgba(229, 228, 146, 255), stop:0.794258 rgba(236, 211, 32, 255))
}
QProgressBar#pb_buildActors:chunk{
	background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:0.5, stop:0.114833 rgba(143, 229, 125, 255), stop:0.794258 rgba(75, 199, 27, 255))
}
QProgressBar#pb_countScenes:chunk{
	background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:0.489, stop:0 rgba(255, 207, 98, 255), stop:0.725389 rgba(255, 138, 21, 255));
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lb_title">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>155</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Phosphate</family>
         <pointsize>120</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Scene Query</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QProgressBar" name="pb_loadScenes">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="value">
          <number>24</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="textDirection">
          <enum>QProgressBar::BottomToTop</enum>
         </property>
         <property name="format">
          <string>%p%</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QProgressBar" name="pb_buildScenes">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="value">
          <number>24</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="textDirection">
          <enum>QProgressBar::BottomToTop</enum>
         </property>
         <property name="format">
          <string>%p%</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Loading Scenes</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QProgressBar" name="pb_buildActors">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="value">
          <number>24</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="textDirection">
          <enum>QProgressBar::BottomToTop</enum>
         </property>
         <property name="format">
          <string>%p%</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Building Actors</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QProgressBar" name="pb_loadActors">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="value">
          <number>24</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="textDirection">
          <enum>QProgressBar::BottomToTop</enum>
         </property>
         <property name="format">
          <string>%p%</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Loading Actors</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Building Scenes</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="Logo">
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>450</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../resources/icons.qrc">:/Icons/SceneQuery Logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
